app/models/manageiq/providers/embedded_ansible/automation_manager/amazon_credential.rb

```diff
@@ -1,28 +1,38 @@
 class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::AmazonCredential < ManageIQ::Providers::EmbeddedAnsible::AutomationManager::CloudCredential
-  COMMON_ATTRIBUTES = {
-    :userid   => {
-      :label     => N_('Access Key'),
-      :help_text => N_('AWS Access Key for this credential'),
-      :required  => true
+  COMMON_ATTRIBUTES = [
+    {
+      :component  => 'text-field',
+      :label      => N_('Access Key'),
+      :helperText => N_('AWS Access Key for this credential'),
+      :name       => 'userid',
+      :id         => 'userid',
+      :required   => true,
```

Comment:
It should be `isRequired` everywhere, according to https://data-driven-forms.org/mappers/component-api#heading-formfieldscomponents#formfieldscomponents